### PR TITLE
Remove "Closes #" from OpenZFS commit messages

### PR DESCRIPTION
**Pull request by dinatale2**
Lines that start with "Closes #" should be removed.

Signed-off-by: Giuseppe Di Natale <dinatale2@llnl.gov>

### CODE REVIEW

**Comment by dinatale2** [comment]
@behlendorf mind giving this a quick review? It's a one liner.

File: scripts/openzfs-merge.sh [hunk 1]
```diff
@@ -103,6 +103,7 @@ generate_desc() {
 	USER_MAIL=$(git config user.email)
 	OPENZFS_COMMIT_AUTHOR=$(git log --format="%aN <%aE>" --remotes=openzfs/master -n 1 $OPENZFS_COMMIT)
 	sed -i '/^$/d' "$TMP_FILE"
+	sed -i '/^Closes #/d' "$TMP_FILE"
```
Comment (behlendorf) [inline scripts/openzfs-merge.sh]: If we're going to automatically remove these lines I'd suggest handling [all keywords](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) which can be used for this.

**Comment by dinatale2** [comment]
Refreshed to include all keywords when they appear at the beginning of a line.

File: scripts/openzfs-merge.sh [hunk 2]
```diff
@@ -103,6 +103,18 @@ generate_desc() {
...
+	# handle github keywords
+	sed -i '/^closes #/Id' "$TMP_FILE"
+	sed -i '/^close #/Id' "$TMP_FILE"
+	sed -i '/^closed #/Id' "$TMP_FILE"
+	sed -i '/^fix #/Id' "$TMP_FILE"
+	sed -i '/^fixes #/Id' "$TMP_FILE"
+	sed -i '/^fixed #/Id' "$TMP_FILE"
+	sed -i '/^resolve #/Id' "$TMP_FILE"
+	sed -i '/^resolves #/Id' "$TMP_FILE"
+	sed -i '/^resolved #/Id' "$TMP_FILE"
```
Comment (gmelikov) [inline scripts/openzfs-merge.sh]: You can use one-liner:
```
sed -i '/^closes #\|^close #\|^closed #\|^fix #\|^fixes #\|^fixed #\|^resolve #\|^resolves #\|^resolved #/Id' "$TMP_FILE"
```

Comment (dinatale2) [inline scripts/openzfs-merge.sh]: I went ahead and broke it down into 3 lines, one for each main keyword. If you wouldn't mind giving it one more look over.

**Pull request merged**